bin/node/src/service.rs

```diff
@@ -190,7 +190,7 @@ fn setup(
                 deny_unsafe,
             };
 
-            Ok(crate::rpc::create_full(deps))
+            crate::rpc::create_full(deps).map_err(Into::into)
```

Comment:
Hmmm, doesn't `?` work here (it should be ok within a closure, shouldn't it?)?

Reply:
Can't do. It fails to deduce the correct type of variable then

Reply:
My bad, forgot `Ok` and error wasn't helpful. Done